src/modules/fetchData.js

```diff
@@ -0,0 +1,39 @@
+async function sendAPI(url, data = {}) {
```

Comment:
- [ ] it would be better if you use the `Arrow function` instead of `function keyword` in order to practice what you've learned and also you will be familiar with both